### ISSUE TITLE
Feature - Helm Dependency Update

### ISSUE THREAD

**Issue by eperdeme** [Praqma/helmsman#231]
Hello,

I'm trying to extend a chart by having a local chart to add a few extra Kubernetes settings (istio config).

The local chart has a requirements.yaml file which specifies the upstream chart dependency. 
During a chart run I can do;
helm dep update
helm install/upgrade ./mychart

However in helmsman I can not find a way to make it do a dep update prior to attempting to install the chart.

helmsman.yaml
```
  dashboard:
    namespace: kube-system
    enabled: true
    chart: ../charts/kube-dashboard
    version: 0.1.0
```

Chart requirements file
```
dependencies:
  - name: kubernetes-dashboard
    version: 1.4.0
    repository: https://kubernetes-charts.storage.googleapis.com/
```

error
```
2019/04/16 11:46:48 Command returned with exit code: . And error message: Error: found in requirements.yaml, but missing in charts/ directory: kubernetes-dashboard
```

**Comment by eperdeme** [comment]
I'll just copy the entire chart and make the modifications direct to a fork of the chart.

**Comment by jc36** [comment]
Faced a similar situation. 
Temporarily solved the problem by manually running the command `helm dependency update` in the chart folder

**Comment by lukipro** [comment]
It would be nice if helmsman could optionally call the ```helm dep up`` step before running the plan. Is there any plan to implement such a feature?

**Comment by sami-alajrami** [comment]
This is implemented in PR #272  ..  @jc36 & @lukipro could you please test using the `--update-deps` with the docker image: `praqma/helmsman:pr-272`

**Comment by sami-alajrami** [comment]
this was shipped in v1.11.0. closing this.
